src/modules/esl-carousel/core/esl-carousel.utils.ts

```diff
@@ -111,12 +100,12 @@ function resolveGroupIndex(indexStr: string | ESLCarouselNavIndex, cfg: ESLCarou
   }
   // TODO: extend navigation boundaries
   if (value === -1 && cfg.activeIndex < cfg.count && cfg.activeIndex > 0) {
-    return {index: 0, direction: direction || 'prev'};
+    return {index: 0, direction: direction || ESLCarouselDirection.PREV};
   }
   if (value === 1 && normalize(cfg.activeIndex + cfg.count, cfg.size) > cfg.size - cfg.count) {
```

Comment:
should we here replace normalize by normalizeIndex?

Reply:
Probably in that check we shouldn't.
That's boundary check for groups so it is respect non-loop carousel boundaries by design. Not sure we should do something with conditions for that.